mmv1/third_party/terraform/services/securitycenter/resource_scc_organization_big_query_export_config_test.go

```diff
@@ -13,13 +18,21 @@ func TestAccSecurityCenterOrganizationBigQueryExportConfig_basic(t *testing.T) {
 	t.Parallel()
 
 	randomSuffix := acctest.RandString(t, 10)
-	dataset_id := "tf_test_" + randomSuffix
+	datasetID := "tf_test_" + randomSuffix
 	orgID := envvar.GetTestOrgFromEnv(t)
 
+	// Run cleanup before the test starts
```

Comment:
Why do we need this? We run sweeper jobs independently. This could cause issues with other tests running at the same time.

Reply:
This is to prevent quota issues, clutter, and resource conflicts in test runs.

Reply:
Let's solve that independently- we'd want the sweepers to work correctly rather than add a separate process. If this runs at the same time as BQ tests are running we'd run into errors as we delete the resources under test.

Reply:
That means we should be fine to remove this block.